src/templates/BuildingDetails.vue

```diff
@@ -185,7 +185,7 @@ query ($id: ID!, $ID: String) {
                 </h2>
 
                 <p>
-                  This building reported zero fossil gas use in the most recent
+                  This building did not report fossil gas use in the most recent
```

Comment:
I tweaked the wording because now it's clearly that we just _don't know_ their gas use, rather than that they strictly reported 0